### DISCUSSION TITLE
Workflow question: How do you create new notes while reading pdfs with pdf++?

### DISCUSSION THREAD

**Discussion by RikardSwed** [RyotaUshio/obsidian-pdf-plus#183] [Q&A]
The plugin is great for making highlights. But sometimes I want to make a note out of a word or concept in the pdf I am reading. How do you go about doing this? At the moment i have no smoothe workflow for this. Perhaps another feature that is needed? Or something that already excists that I dont know of?

**Comment by N3C2L** [comment]
Macros are in the pipeline: 
https://github.com/RyotaUshio/obsidian-pdf-plus/issues/159
With this implemented, you can then extract fresh notes from highlights. For example, after highlighting and clicking a button or in the context menu. 
Macros are currently available in this three plugins: QuickAdd, Commander, Better command palette.
But the macros can't be used for PDF files. So that's why @RyotaUshio will make us happy with a macro function for PDF files 😀.

**Comment by RikardSwed** [comment]
Sounds great. Thank you for the info
